notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>Have you tried to see what happen with the positioning when doing only a azimuth slew for a fixed elevation? In principle it should be easier to keep the position in that case, but have you tried that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I think that Craig did this analysis</p>